table/block_based/block_based_table_reader.cc

```diff
@@ -2910,7 +2910,7 @@ void BlockBasedTableIterator<TBlockIter, TValue>::InitDataBlock() {
           if (!rep->file->use_direct_io() &&
               (data_block_handle.offset() +
                    static_cast<size_t>(data_block_handle.size()) +
-                   kBlockTrailerSize >
+                    kBlockTrailerSize >
```

Comment:
How about `block_size(data_block_handle)` here?

Reply:
The definition is not available here (BlockBasedTableIterator). Maybe a follow-on PR. It'd be good to use a helper function in all places where kBlockTrailerSize is referenced.

Reply:
`block_size` could be a public helper in `BlockBasedTable`, right? On the other hand, if you're going to leave this as-is, could you please revert the whitespace change?

Reply:
I like the public helper idea. It could be in format.h and available to all.